components/atoms/SelectField.js

```diff
@@ -37,7 +37,7 @@ export function SelectField(props) {
         ) : undefined}{" "}
         {props.label}{" "}
         {props.required ? (
-          <b className="text-error-border-red" aria-hidden="true">
+          <b className="text-error-border-red sr-only" aria-hidden="true">
```

Comment:
Having both `sr-only` and `aria-hidden="true"` here makes this invisible to both screen readers and regular users. I've somewhat flip flopped on this but after some thought, I think given that we have `aria-required` on the input itself and seeing that most popular screen readers have support for `aria-required`, I'd say we just get rid of this line (and any red `required` on the other components).